LLama.Unittest/IgnoreOnCIFact.cs

```diff
@@ -0,0 +1,22 @@
+namespace LLama.Unittest;
+
+/// <summary>
```

Comment:
We could probably add a symbol into the build in CI, so you can detect it directly?